powa/static/js/components/Grid.vue

```diff
@@ -208,14 +206,12 @@ function getFormatter(type) {
       return (value) => formatDuration(value, true);
     case "percent":
       return (value) => value + "%";
-    case "query":
-      return formatQuery;
     case "size":
       return formatSize;
     case "integer":
       return (value) => value.toLocaleString();
     default:
-      return (value) => value;
+      return (value) => _.escape(value);
```

Comment:
Can you please confirm that this is really required? If so, this need some explanations.

Reply:
it avoids having 3 cases in the gridCellFormatter, for example with the <local> server name when put in v-html it is interpreted as html and a closing tag </local> is added making the name not visible. i'll update the commit message.